python/surf/devices/linear/_Ltc3815.py

```diff
@@ -0,0 +1,100 @@
+#-----------------------------------------------------------------------------
+# This file is part of the 'SLAC Firmware Standard Library'. It is subject to
+# the license terms in the LICENSE.txt file found in the top-level directory
+# of this distribution and at:
+#    https://confluence.slac.stanford.edu/display/ppareg/LICENSE.html.
+# No part of the 'SLAC Firmware Standard Library', including this file, may be
+# copied, modified, propagated, or distributed except according to the terms
+# contained in the LICENSE.txt file.
+#-----------------------------------------------------------------------------
+
+import pyrogue as pr
+
+import surf.protocols.i2c
+
+class Ltc3815(surf.protocols.i2c.PMBus):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+        self.add(pr.LinkVariable(
+            name         = 'Vin',
+            mode         = 'RO',
+            units        = 'V',
+            typeStr      = "Float32",
+            disp         = '{:1.3f}',
+            linkedGet    = lambda: self.READ_VIN.value()*4.0E-3, # Conversion factor: 4mV/Bit
```

Comment:
This should be `lambda read: self.READ_VIN.get(read=read)`. Same for all the others.

This allows rogue to do a hardware fetch when you call `self.Vin.get(read=True)` in a script.

Reply:
Fixed in [345b245](https://github.com/slaclab/surf/pull/1158/commits/345b245b83fbd3dd6ea0244375ccec4a3d28531e)